PhysicsTools/NanoAOD/python/taus_updatedMVAIds_cff.py

```diff
@@ -534,15 +538,273 @@
+    mvaName_wGwGSF_BL = 'RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_wGwGSF_BL',
+    mvaName_wGwGSF_EC = 'RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_wGwGSF_EC',
+    mvaName_woGwGSF_BL = 'RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_woGwGSF_BL',
+    mvaName_woGwGSF_EC = 'RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_woGwGSF_EC'
+)
+## anti-e v1 WPs
+# VLoose
+patTauDiscriminationByVLooseElectronRejectionMVA62015 = patTauDiscriminantCutMultiplexer.clone(
+    PATTauProducer = patTauDiscriminationByElectronRejectionMVA62015Raw.PATTauProducer,
+    Prediscriminants = patTauDiscriminationByElectronRejectionMVA62015Raw.Prediscriminants,
+    toMultiplex = cms.InputTag("patTauDiscriminationByElectronRejectionMVA62015Raw"),
+    key = cms.InputTag("patTauDiscriminationByElectronRejectionMVA62015Raw","category"),
+    mapping = cms.VPSet(
+        cms.PSet(
+            category = cms.uint32(0),
+            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_NoEleMatch_woGwoGSF_BL_WPEff99'),
+            variable = cms.string('pt')
+        ),
+        cms.PSet(
+            category = cms.uint32(2),
+            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_NoEleMatch_wGwoGSF_BL_WPEff99'),
+            variable = cms.string('pt')
+        ),
+        cms.PSet(
+            category = cms.uint32(5),
+            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_woGwGSF_BL_WPEff99'),
+            variable = cms.string('pt')
+        ),
+        cms.PSet(
+            category = cms.uint32(7),
+            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_wGwGSF_BL_WPEff99'),
+            variable = cms.string('pt')
+        ),
+        cms.PSet(
+            category = cms.uint32(8),
+            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_NoEleMatch_woGwoGSF_EC_WPEff99'),
+            variable = cms.string('pt')
+        ),
+        cms.PSet(
+            category = cms.uint32(10),
+            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_NoEleMatch_wGwoGSF_EC_WPEff99'),
+            variable = cms.string('pt')
+        ),
+        cms.PSet(
+            category = cms.uint32(13),
+            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_woGwGSF_EC_WPEff99'),
+            variable = cms.string('pt')
+        ),
+        cms.PSet(
+            category = cms.uint32(15),
+            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_wGwGSF_EC_WPEff99'),
+            variable = cms.string('pt')
+        )
+    )
+)
+# Loose
+patTauDiscriminationByLooseElectronRejectionMVA62015 = patTauDiscriminationByVLooseElectronRejectionMVA62015.clone(
+    mapping = cms.VPSet(
```

Comment:
can't all of the below be rolled up in a simple loop? smth like
```python
patTauDiscriminationByLooseElectronRejectionMVA62015 = patTauDiscriminationByVLooseElectronRejectionMVA62015.clone()
for m in patTauDiscriminationByLooseElectronRejectionMVA62015.mapping : 
    m.cut = m.cut.replace("WPeff99", "WPeff96")
```

Reply:
Indeed, why not. Update follows.

Reply:
cms.string does not seem to support replace or conversion to python string. Do you have an idea, how to do this in an elegant way?

Reply:
if it gets too ugly, we can also wait for the new tau discriminator format, where these WPs and categories will be separated in the configurations and then 'multiplied' by the module. https://github.com/swozniewski/cmssw/blob/restructure_PFTauDiscriminators_v3/RecoTauTag/Configuration/python/HPSPFTaus_cff.py#L320-L376

Reply:
.pythonValue may be needed on rhs (before .replace)
https://github.com/cms-sw/cmssw/blob/master/FWCore/ParameterSet/python/Types.py#L300
@Dr15Jones @makortel

Reply:
Thanks for the hint, this could be used! However, this function puts the string in single quotes, which I have to remove:
`m.cut = cms.string(m.cut.configValue().strip("'").replace("WPeff98", "WPeff90"))`
Does not look very nice but would work...

Reply:
there is also a .configValue return.

Reply:
Use `m.cut.value()` which returns the held python string.  Remember, you can use python to look at any of the classes to find what methods they have
```bash
bash>python
>>> import FWCore.ParameterSet.Config as cms
>>> help(cms.string)
```

Reply:
Thanks! This looks good